test_pipeline.py

```diff
@@ -81,9 +84,7 @@ def test_pipeline(transcription_id = "abc"):
             assert updates[0].state == "transcoding"
 
             assert type(updates[1]) == TranscodingProgress
-            assert updates[2].percent_done == 96
-            assert updates[5].percent_done is 100
-            assert updates[5].track.duration > 1.4
+            assert updates[4].track.duration > 1.4
```

Comment:
these assertions seem highly cpu specific. thought maybe you were running these on modal to get consistent results, but didn't see how

Reply:
yep flappy